File: Menu Driven Class Editor/Person.java
public class Person {
	public String name;
	public String address;
	public String phone;
	Person(String name, String address, String phone) {
		this.name = name;
		this.address =  address;
		this.phone = phone;
	}
	void setName(String n) {
		name = n;
	}
	public String getName() {
		return name;
	}
	public String getPhone() {
		return phone;
	}
	public String toString() {
		return "Name: " + name + "\nAddress: " + address + "\nPhone: " + phone;
	}
}

File: Linked List Inheritance/Person.py
class Person:
    def __init__(self,newName='none',address='none',phone='999-999-9999'):
        self.name = newName
        self.address = address
        self.phone = phone
    
    def setName(self,stringOfName):
        self.name = stringOfName
    
    def getName(self):
        return self.name
    
    def getPhone(self):
        return self.phone
    
    def __str__(self):
        return f'Name: {self.name}\nAddress: {self.address}\nPhone: {self.phone}'
    
    